external-crates/move/crates/move-cli/src/base/new.rs

```diff
@@ -20,6 +22,8 @@ pub const MOVE_STDLIB_ADDR_VALUE: &str = "0x1";
 
 /// Create a new Move package with name `name` at `path`. If `path` is not provided the package
 /// will be created in the directory `name`.
+///
+/// By default, this command allows a strict naming scheme based on this regex: [A-Za-z][A-Za-z0-9-_]*, and it will replace hyphens (-) with underscore (_) in the address field in the Move.toml file.
```

Comment:
This comment is a bit weird for the move-cli side since it does not generate an address

Reply:
Is there a space to move this to the sui-move side of things?

Reply:
You're right. Let me try to move it and see how it goes!